### PR TITLE
Add CHANGELOG for 5.15.0 release

### PR DESCRIPTION
**Pull request by hush-hush**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,41 @@
 Changes
 =======
 
+# 5.15.0 / 07-10-2017
```
Comment (truthbk) [inline CHANGELOG.md]: This should be `UNRELEASED` instead of a date. Once we release we update the changelog.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,41 @@
...
+**Linux, Windows, Docker and Source Install**
+
+### Details
+https://github.com/DataDog/dd-agent/compare/5.14.1...5.15.0
+
+### Changes
+Please refer to the [Integrations Core Checks 5.15.0 tag](https://github.com/DataDog/integrations-core/releases/tag/5.15.0) for the list of changes on the core checks.
+Please refer to the [Trace Agent 5.15.0 tag](https://github.com/DataDog/datadog-trace-agent/releases/tag/5.15.0) for the list of changes on the Trace Agent.
+
+* [FEATURE] [kube] Allow to run agent on the node [#3401][]
```
Comment (truthbk) [inline CHANGELOG.md]: Are you using `pimpmychangelog`? It automatically handles the PR and mentions to contributors - I don't see the links at the bottom of the changelog.md file so I'm assuming not. It's a gem super easy to install... grab it here: https://github.com/pcreux/pimpmychangelog :)

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,41 @@
...
+* [FEATURE] Add k8s 1.6 to cgroup parsing dockerutil test [#3377][]
+* [FEATURE] Mesos tagging and generic container tagger [#3375][]
+* [FEATURE] [kubernetes] add kubernetes container name [#3368][]
+* [FEATURE] Add support for etcd auth [#3357][]
+* [FEATURE] Add support for %%id%% and %%pid%% template variables [#3404][]
+* [IMPROVEMENT] Add container name tag in auto discovery [#3421][]
+* [IMPROVEMENT] Mesos: extract chronos_job and chronos_job_owner if not empty [#3417][]
+* [IMPROVEMENT] Docker: disable mesos_task and chronos_job tags because of high cardinality [#3414][]
+* [IMPROVEMENT] Disable kube_service tagging on 4 403 errors from apiserver [#3413][]
+* [IMPROVEMENT] constant for container name label [#3405][]
```
Comment (truthbk) [inline CHANGELOG.md]: We always thank community contributions. So for example this one should also say: `thanks @_foo`

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,41 @@
...
+* [IMPROVEMENT] try collecting docker+orchestrator host tags even if not containerised [#3403][]
+* [IMPROVEMENT] Update Mac restart command [#3390][]
+* [IMPROVEMENT] [docker] Add kube/ecs/nomad host tags + port to BaseUtil [#3386][]
+* [IMPROVEMENT] Try both 4001 and 2379 etcd ports in SD if not configured [#3351][]
+* [SANITY] Moves common wait_for_async network function out to the Agent [#3408][]
+* [SANITY] [util] Move `Timer` to its own `utils.timer` module [#3391][]
+* [SANITY] [checks] Deprecate `AgentCheck.set` [#3385][]
+
```
Comment (truthbk) [inline CHANGELOG.md]: Missing this one: https://github.com/DataDog/dd-agent/pull/3364

File: CHANGELOG.md (hunk 2 above)
Comment (hush-hush) [inline CHANGELOG.md]: :+1:

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,42 @@
 Changes
 =======
 
+# 5.15.0 / UNRELEASED
+**Linux, Windows, Docker and Source Install**
```
Comment (truthbk) [inline CHANGELOG.md]: Also, maybe mac.... but this is just a note for reference.

**Pull request merged**